davenmathews/Bright_Events
# Implement filter event by category and location

## This pull request implements event search

This is on top of other previous commits

**The tasks completed are**
GET /api/v1/events/<string:category>

**This can be manually tested by:**
1. Clone the repository.
2. Use virtualenv to start a virtual environment.
3. Ensure the users.json and events.json are placed in your virtualenv folder and that they are empty
3. Install the requirements.txt.
4. Run appv1.py and open the local host in your browser.
5. Register new users and log in.
6. Create multiple events with different categories
7. Click on the categories links to filter them

**Constrains:**
Will not create event if date given is less than today's date

**This finishes filter events by location because the user can enter a location on search bar and get events that have their location set to the query parameter**

**[Finishes|Delivers #152972666 #152972624]**
